src/cljs/pyregence/config.cljs

```diff
@@ -607,12 +607,26 @@
                                                               [:br]
                                                               [:br]
                                                               [:strong "GridFire"]
-                                                              " is a fire behavior model developed by Gary Johnson of Spatial Informatics Group. It combines empirical equations from the wildland fire science literature with the performance of a raster-based spread algorithm using the method of adaptive time steps and fractional distances."]
-                                                 :options    {:elmfire  {:opt-label "ELMFIRE"
-                                                                         :filter    "elmfire"}
-                                                              :gridfire {:enabled?  #(feature-enabled? :gridfire)
-                                                                         :opt-label "GridFire"
-                                                                         :filter    "gridfire"}}}
+                                                              " is a fire behavior model developed by Gary Johnson of Spatial Informatics Group. It combines empirical equations from the wildland fire science literature with the performance of a raster-based spread algorithm using the method of adaptive time steps and fractional distances."
+                                                              [:br]
+                                                              [:br]
+                                                              [:strong "Pyretechnics"]
+                                                              " is a Python"
+                                                              [:a {:href   "https://github.com/pyregence/pyretechnics/ "}
+                                                                " open source"]
+                                                              [:a {:href   "https://pypi.org/project/pyretechnics/"}
```

Comment:
Let's attach this link to "open source library". Can you please add `:target "_blank"` to the link?